src/app/gallery/components/public-archives/public-archives.component.scss

```diff
@@ -0,0 +1,108 @@
+@import 'variables';
+
+.public-archive {
+  display: flex;
+  font-family: Arial, sans-serif;
+  background: #fff;
+  padding: 20px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  &-info {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.public-archives-list {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  max-height: 320px;
+  overflow: scroll;
```

Comment:
In the design I only see vertical scrollbars (and no scrollbars in mobile view). I think that at the very least this should have either
```css
overflow-x: auto;
/* OR */
overflow-x: hidden;
```
rules enabled so that the horizontal scrollbars disappear. It isn't clear in the design if we want scrollbars always visible even if there are not enough archives in the list to result in scrollbars; we should get some clarification on that from the experience team.

Reply:
@crisnicandrei I don't think we want horizontal scrollbars (which I'm not seeing) and probably don't want the scrollbars if there's not enough archives in the list to result in scrollbars. I also think the scrollbar on mobile would be a little hard to use. I think the design for mobile shows a "see more" after the first three archives, that would expand to a list of all your public archives. Could we make that change?